tasks/GPSDTask.cpp

```diff
@@ -1,4 +1,14 @@
-#include "GPSDTask.hpp"
+
+// workaround for name clash in 
+// usr/include/mach/policy.h and gpsd/3.15/include/gps.h
```

Comment:
I am *so* missing something. Why do you need to define policy_t before including GPSDTask.hpp if the problem is in libgpsmm ?